tests/languages/php/test.php

```diff
@@ -118,6 +118,11 @@
 );
 echo $url . "\n";
 
+$response = $general->multipart();
+echo "{$response['x']}\n";
+$hash = md5($response['responseBody']->toBinary());
+echo "{$hash}\n";
+
```

Comment:
cool this is good, but this is an example of how we receive multipart data right? Don't we meed to have a test for  sending multipart as well ?

Reply:
We already have a test for this, the `upload` function